### PR TITLE
Rpmostree bootloader

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by cgwalters** [comment]
This is a potential fix for https://github.com/rhinstaller/anaconda/issues/900

I'm submitting this for pre-review, but I haven't yet tested it significantly.  The real fix was in https://github.com/rhinstaller/anaconda/pull/903

**Comment by jkonecny12** [comment]
Jenkins, it's ok to test.

File: pyanaconda/bootloader.py [hunk 1]
```diff
@@ -2460,7 +2459,7 @@ def writeBootLoader(storage, payload, instClass, ksdata):
     if hasattr(storage.bootloader, 'efi_dir'):
         storage.bootloader.efi_dir = instClass.efi_dir
 
-    if isinstance(payload, RPMOSTreePayload):
+    if payload.handlesBootloaderConfiguration():
```
Comment (rvykydal) [inline pyanaconda/bootloader.py]: Seems like a typo - calling the value returned by the property.

**Comment by rvykydal** [comment]
@cgwalters 
Colin, do you still want to merge the change?

**Comment by InStallker** [comment]
Can one of the admins verify this patch?

**Comment by InStallker** [comment]
Can one of the admins verify this patch?

**Comment by M4rtinK** [comment]
@cgwalters Ping! ;-)

**Comment by cgwalters** [comment]
Closing this, it probably still makes sense but I am not aware of a driver for fixing it.

**Pull request closed without merging**